CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by a [Code of Conduct](code_of_conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [quack@duckdb.org](mailto:quack@duckdb.org).
+
+## Eligibility
+
+Before submitting a contribution, please check whether your contribution is eligible.
+
+1. Before creating a new page, please do a search for the proposed page in the [existing documentation](https://duckdb.org/docs/).
```

Comment:
Maybe just direct people to use GitHubs search?

Reply:
Even this: https://duckdb.org/docs/search